### PR TITLE
Feat: create data model

### PR DESCRIPTION
**Pull request by Hope1226**
In this feature branch I:

- [x] Built the project's schema.
- [x] Created necessary migration files based on the ERD diagram.
- [x] Foreign keys have been included.
- [x] All columns that are foreign keys have been set with a corresponding index.

### CODE REVIEW

File: app/models/like.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :comment
```
Comment (ahmetbozaci) [inline app/models/like.rb:3]: - [ ] Like table should belong to post. Please fix it.

[ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
+  belongs_to :user
+end
```
Comment (ahmetbozaci) [inline app/models/post.rb:3]: - [ ] The post table has many likes and comments. Please fix it.

[ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```
Comment (ahmetbozaci) [inline app/models/user.rb:2]: - [ ] The user table has many likes, posts, and comments. You should add it.

[ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Hope1226  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.


___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @Hope1226 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**